.coderabbit.yaml

```
@@ -0,0 +1,36 @@
+coderabbit:
+  # Enable code analysis
+  code_review:
+    enabled: true
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)